web-ui/src/main/resources/catalog/components/common/ows/OWSService.js

```diff
@@ -81,19 +82,14 @@
           var url = result.Capability.Request.GetMap.
               DCPType[0].HTTP.Get.OnlineResource;
 
+
           // Push all leaves into a flat array of Layers
           // Also adjust crs (by inheritance) and url
-          var getFlatLayers = function(node, parentCrs) {
-            // current inherited crs (w/o dupes)
-            var crs = (parentCrs || []).concat(node.CRS || [])
-                .filter(function(value, index, array) {
-                  return array.indexOf(value) === index;
-                });
-
+          var getFlatLayers = function(node) {
```

Comment:
> GetCap object given by OL is missing some CRS

So this should be fixed on OL side no ? Is that an option ?

Reply:
Since we're not up to date with the latest OL version, I think this is not really an option for us. I'll try to see if the issue still exist in the latest version and create an issue on their side if it does.